scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -47,6 +49,11 @@ class ThemeAggregator {
   static aggregateTheme(themeName, options = {}, defaultFlag) {
     const { theme, scoped = [] } = options;
 
+    if (!themeName) {
+      Logger.warn(`Failed to aggregate ${themeName}. Falsey values are not accepted.`);
```

Comment:
won't this log:
```
Failed to aggregate undefined. Falsey values are not accepted.
```

Reply:
we can add quotes around themeName? i'd like to log the value for consumers to pinpoint what is incorrect